src/Nodes/IoTHub/IoTHubAccountTreeItem.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
It's good to have a folder dedicated for IoTHub tree items. Please also help to move the same set of DPS treeitems to a new DPS folder?

Reply:
Moved